### DISCUSSION TITLE
<Foundations>: <Another method to creating a boilerplate>

### DISCUSSION THREAD

**Discussion by CBID2** [TheOdinProject/curriculum#25665] [Ideas]
### Describe your suggestion

I was reading through the Shortcut paragraph of the boilerplate, and thought I'd add another method. When the VSCode shortcut does not work, I type the following in my blank HTML file:
`html:5`. 
I think this would help beginners as it's easier to remember.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-html-boilerplate

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### (Optional) Discord Name

Chrissy

### (Optional) Additional Comments

This is my first time being in this repo. I look forward to contributing to it! :)

**Comment by KevinMulhern** [comment]
Hey @CBID2, thanks for taking the time to suggest this. I moved it to a discussion for now. We don't feel like adding another shortcut here would be useful at this time. But we'll leave this open to gather more opinions.

**Reply by CBID2** [reply]
> Hey @CBID2, thanks for taking the time to suggest this. I moved it to a discussion for now. We don't feel like adding another shortcut here would be useful at this time. But we'll leave this open to gather more opinions.

Thanks @KevinMulhern. I raised an issue about the [commit message lesson](https://github.com/TheOdinProject/curriculum/issues/25664)
